content/tutorials/build-a-data-source-backend-plugin.md

```diff
@@ -89,6 +89,14 @@ You now have a new data source instance of your plugin that is ready to use in a
 1. A line graph is rendered with one series consisting of two data points.
 1. Save the dashboard.
 
+### Troubleshooting
+
+#### Does the list of available data sources not contain your plugin?
+
+As stated on the [plugin signature verification documentation](https://grafana.com/docs/grafana/latest/plugins/plugin-signature-verification/#backend-plugins):
```

Comment:
- Avoid leading with the reference.
- Use contractions, e.g. "isn't" instead of "is not".
- When linking to a document, use the heading of that document as link text.

Instead:

> By default, Grafana requires backend plugins to be signed. To load unsigned backend plugins, you need to
configure Grafana to [allow unsigned plugins](https://grafana.com/docs/grafana/latest/plugins/plugin-signature-verification/#allow-unsigned-plugins). For more information, refer to [Plugin signature verification](https://grafana.com/docs/grafana/latest/plugins/plugin-signature-verification/#backend-plugins).

Reply:
Thanks for your feedback & hints. I really appreciate them 😍